src/store/BarrierStore.js

```diff
@@ -4,7 +4,6 @@ import ShadeStore from './ShadeStore';
 import PendingPromise from '../utils/PendingPromise';
 
 export default class BarrierStore {
-    static get SHADE_COLOR_RED() { return 'red'; }
     static get SHADE_COLOR_GREEN() { return 'green'; }
```

Comment:
Just remove this as well. In `Barrier.jsx` you can use default params to set `green` (like how `color` prop is set